src/initialize.rs

```diff
@@ -769,88 +751,18 @@ mod tests {
         #[test]
         fn destination_is_correct_with_output() {
             let expected = PathBuf::from("output");
-            let mut e = Execution::default();
-            e.output = Some(expected.clone());
-            let actual = e.destination().unwrap();
-            assert_eq!(actual, expected);
-        }
-
-        #[test]
-        fn destination_is_correct_with_input() {
```

Comment:
All those tests are irrelevant now, since destination is always relative to the provided package.